packages/kit/src/views/Send/pages/SendDataInput/SendDataInputContainer.tsx

```
@@ -1189,7 +1206,7 @@ function SendDataInputContainer() {
     if (isNFT) {
       return renderNFTDataInputForm();
     }
-    if (displayAmountFormItem) {
+    if (true) {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)